tools/validators/instance_validator/error_guide.md

```diff
@@ -0,0 +1,1712 @@
+# Instance Validator Error Message Guide
+Please use the following documentation to lookup descriptions and examples of any instance validation errors resulting from your config file.
+
```

Comment:
Suggest adding a table of contents here -- you can quickly link it to the correct section in the file like this 

- [A file section](#the-name-of-the-section-header)